python/cugraph/tests/test_betweenness_centrality.py

```diff
@@ -55,7 +55,7 @@
 # Comparison functions
 # =============================================================================
 def calc_betweenness_centrality(
-    graph_file,
+    Fixture_params_dts,
```

Comment:
This seems like it could be a little confusing since pytest won't call this as a fixture like it would for a test. Maybe this could be named something like `graph_obj_tuple` or something that describes it's the tuple of graph objects passed in?

Reply:
I renamed it `graph_obj_tuple`